src/components/AddItems.js

```diff
@@ -36,6 +42,24 @@ const AddItemsToList = () => {
       .get()
       .then((data) => {
         if (data.docs.length) {
+          const { items } = data.docs[0].data();
+          const nonWordCharactersAndUnderscores = /[\W_]/g;
+          const shoppingListItemExists = items.some(
+            (shoppingListItemObject) => {
+              return (
+                shoppingListItemObject.shoppingListItemName
+                  .toLowerCase()
+                  .replace(nonWordCharactersAndUnderscores, '') ===
+                shoppingListItemName
+                  .toLowerCase()
+                  .replace(nonWordCharactersAndUnderscores, '')
+              );
+            },
+          );
+          if (shoppingListItemExists === true) {
```

Comment:
Since the `some()` method always returns a boolean you can omit the `=== true` here and just write `if (shoppingListItemExists)`

Reply:
Yes, I'd second Golfo here. 👍